### DISCUSSION TITLE
Feature request: Show "commit message" in the "Versions" dialog

### DISCUSSION THREAD

**Discussion by pchemguy** [zim-desktop-wiki/zim-desktop-wiki#2031] [Ideas]
Please add "commit message" column in the "Versions" dialog. Also, I do not see what is the use for the "Rev" column with hashes.

**Comment by introt** [comment]
Just needs a

```python3
            (_('Comment'), self.MSG_COL), # T: Column header versions dialog
```

here:

https://github.com/zim-desktop-wiki/zim-desktop-wiki/blob/85b6d81d31652042b1e9b007b211f9aab435047b/zim/plugins/versioncontrol/__init__.py#L1063-L1067

I'd also reorder the columns, and switch this out for `MSG_COL`:

https://github.com/zim-desktop-wiki/zim-desktop-wiki/blob/85b6d81d31652042b1e9b007b211f9aab435047b/zim/plugins/versioncontrol/__init__.py#L1074-L1075

...and set the default ordering to use the timestamp, but that one seems to be a bit more tricky.

The revision hashes would be useful when using the version control system's command line interface if only they were copyable - that's my feature request.
